mlx/traceability.py

```diff
@@ -395,38 +396,43 @@ def query_checklist(settings, attr_values):
     Returns:
         (dict) The query results with zero or more key-value pairs in the form of {item ID: ItemInfo}.
     """
-    query_results = {}
     headers = {}
     if not settings.get('private_token'):
         settings['private_token'] = ''
-    if 'github' in settings['api_host_name']:
+    git_platform = settings['git_platform'].lower() if 'git_platform' in settings else settings['api_host_name']
```

Comment:
this oneliner will keep me awake at night for sure... but i guess that is pythonic.